common/shlibs

```diff
@@ -3980,3 +3980,4 @@ libselinux.so.1 libselinux-3.0_1
 libsepol.so.1 libsepol-3.0_1
 libfrrcares.so.0 libfrr-7.3.1_1
 libhugetlbfs.so.0 libhugetlbfs-2.22_1
+libvkbasalt.so vkBasalt-0.3.2.1_1
```

Comment:
Do any applications link against this? Or is it just dlopened at runtime? If the latter, it _probably_ doesn't need to be here. I would wait for confirmation from someone else, though.

Reply:
Doesn't seem like anything links against that, out of maybe three other programs that use vkBasalt. Thanks, will probably be removed